R/Sound_DMR.R

```diff
@@ -622,10 +622,10 @@ run_model <- function(data, i, Output_Frame, formula, model_type,
 #' the threshold. Both the methylated and unmethylated samples need a read depth
 #' greater than or equal to this threshold in order to be considered for the model.
 #' @inheritParams subset_cols
-#' @return Output_Frame (df) - Output_Frame with the summary statistics from the model
+#' @return methyl_summary (df) - Output_Frame with the summary statistics from the model
```

Comment:
The individual_dmr function uses the variable name "methylsummary", maybe change here for consistency?